Mendelugram/Mendelugram/View/GridPhoto/PhotoCollectionViewCell.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  PhotoCollectionViewCell.swift
+//  Mendelugram
+//
+//  Created by Pacek on 21/04/2019.
+//
+
+import UIKit
+
+class PhotoCollectionViewCell: UICollectionViewCell {
+
+    static let reuseIdentifier = String(describing: PhotoCollectionViewCell.self)
```

Comment:
If you want to have it like this for sake of simplicity then no problem but it would be nicer if you defined this in an UICollectionViewCell extension or a protocol with protocol extenstion